src/index.html

```diff
@@ -17,11 +17,481 @@
+    <!--#region Header-->
+    <header class="header">
+      <div class="header__logo">
+        <a
+          class="header__logo-link logo-1200"
+          href="index.html"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!--#endregion Header-->
+    <main class="catalog">
+      <!--#region CARD-->
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__content">
+          <div class="card__image"></div>
+
+          <div class="card__title">
+            <p class="card__title-text">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+          </div>
+
+          <div class="card__code">
+            <p class="card__code-text">Product code: 195434</p>
+          </div>
+
+          <div class="card__rating">
+            <div class="card__rating-stars">
+              <div class="card__rating-star"></div>
+              <div class="card__rating-star"></div>
+              <div class="card__rating-star"></div>
+              <div class="card__rating-star"></div>
+              <iv class="card__rating-star"></iv>
```

Comment:
There is a typo in your HTML tag. It should be 'div' instead of 'iv'. This will cause your HTML to not render correctly.